ci/mg/build.sh

```diff
@@ -22,10 +22,10 @@ export PATH=/conda/bin:/usr/local/cuda/bin:$PATH
 export PARALLEL_LEVEL=4
 export CUDA_REL=${CUDA_VERSION%.*}
 
-# Parse git describe
```

Comment:
I think you have a merge issue, because this change is undoing a recent fix. I see this issue in several files.

Reply:
Let me try and see if I can fix this